libs/design/media-gallery/examples/src/examples.ts

```diff
@@ -0,0 +1,10 @@
+import { BasicMediaGalleryComponent } from './basic-media-gallery/basic-media-gallery.component';
+import { BasicMediaGalleryModule } from './basic-media-gallery/basic-media-gallery.module';
+
+export const MEDIA_GALLERY_EXAMPLES = [
+  BasicMediaGalleryComponent,
+];
+
+export const MEDIA_GALLERY_EXAMPLES_MODULES = [
```

Comment:
Take a look at how I structured the callout examples, I think we don't need to expose these modules like this.

Reply:
changed